pallets/common/src/traits/multisig.rs

```diff
@@ -70,6 +70,9 @@ decl_event!(
         ProposalExecutionFailed(DispatchError),
         /// Scheduling of proposal fails.
         SchedulingFailed(DispatchError),
+        /// Event emitted when there's an error in proposal execution
+        /// Arguments: caller DID, multisig, proposal ID, error.
+        PoposalFailedToExecute(IdentityId, AccountId, u64, DispatchError),
```

Comment:
This was requested on MESH-2118, but I'm not sure we need it. From what I see, if a proposal fails, two events are emitted `ProposalExecutionFailed` and `ProposalExecuted` the second event has the information requested.

Reply:
I don't think we need this event then.  But we should improve the description of the `ProposalExecuted` event, to explain that it is emitted for both success and failure.  @F-OBrien Do the two events provide what is needed?